assignment09/part02/index.php

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 2</title>
+</head>
+<body>
+    
+    <?php
+       
+        $my_favorite_things = array ("Phone","Pen", "Plants", "Apples"  );
+        
+    ?>
+
+    <h1>My Favorite Things</h1>
+
+     <?php   
+     $counter=0;    
+     while ($counter < count($my_favorite_things)){
+        echo ($counter+1).".". $my_favorite_things[$counter]."</br>";
```

Comment:
nice while loop! And you are outputting each element of the array - but we are missing the ordered list. I would expect it to open (<ol>) before the loop and close (</ol>) after the loop with an <li> for each element inside the loop.